CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Static char arrays weren't accepted for `'%s'` in `FormatArgumentType`.
+- Obscure grammar bug where imaginary tokens could technically be matched in valid source code.
```

Comment:
It might be better to keep this entry in the same format as the others?
It is true it was obscure, but it's not really relevant to the change itself.

Thoughts on something like "Lexer incorrectly identifying imaginary tokens in valid source code."?